application/Profile/DBProfile.py

```diff
@@ -188,6 +219,8 @@ def query(self):
                 "username": session.username
             }
 
+        logger.info("Query user sessions successful")
```

Comment:
1. Can we make this a `debug`-level log?
2. Can we log the `_profile`? To stringify the object, we can use `json.dumps()`.